src/util/drawCircle.js

```diff
@@ -1,3 +1,4 @@
+// @deprecated Use drawing.js:drawCircle()
```

Comment:
Could you use jsdoc comments (http://usejsdoc.org/about-getting-started.html#adding-documentation-comments-to-your-code)? I think it needs /* instead. That way this function will start to get a strikethrough in people's code editors (e.g. WebStorm).

Reply:
Worth noting, some editors have plugins that allow you to right click and autogenerate `jsdoc` stubs. IE.https://marketplace.visualstudio.com/items?itemName=joelday.docthis

Reply:
Done.